### PR TITLE
Make copy changes to setup wizard

### PR DESCRIPTION
**Pull request by donnapep**
#### Changes proposed in this Pull Request:

This PR carries over the copy changes to the wizard setup screens that were made in Calypso.

#### Testing instructions:

Click through the setup wizard and ensure the copy matches what is used in Calypso. Click the links to ensure you're taken to the proper page.

#### Screenshots

_Step 1_

screen shot 2017-09-29 at 12 05 53 pm

_Step 2_

screen shot 2017-09-29 at 12 06 46 pm

_Step 3_

screen shot 2017-09-29 at 12 15 17 pm

### CODE REVIEW

File: includes/admin/class-wp-job-manager-setup.php [hunk 1]
```diff
@@ -189,26 +190,22 @@ public function output() {
...
 								<td><code>[submit_job_form]</code></td>
 							</tr>
 							<tr>
 								<td><input type="checkbox" checked="checked" name="wp-job-manager-create-page[job_dashboard]" /></td>
 								<td><input type="text" value="<?php echo esc_attr( _x( 'Job Dashboard', 'Default page title (wizard)', 'wp-job-manager' ) ); ?>" name="wp-job-manager-page-title[job_dashboard]" /></td>
 								<td>
-									<p><?php _e( 'This page allows employers to manage and edit their own jobs from the front-end.', 'wp-job-manager' ); ?></p>
-
-									<p><?php _e( 'If you plan on managing all listings from the admin dashboard you can skip creating this page.', 'wp-job-manager' ); ?></p>
+									<p><?php _e( 'Creates a page that allows employers to manage their job listings directly from a page on your website, instead of requiring them to log in to an admin area. If you want to manage all job listings from the admin dashboard only, you can uncheck this settin', 'wp-job-manager' ); ?></p>
```
Comment (jom) [inline includes/admin/class-wp-job-manager-setup.php]: I think the string was cut off here.

**Review by jom — CHANGES_REQUESTED** [review]
One minor fix. Huge improvement!

File: includes/admin/class-wp-job-manager-setup.php (hunk 1 above)
Comment (donnapep) [reply includes/admin/class-wp-job-manager-setup.php]: Right you are! How's about this - https://github.com/Automattic/WP-Job-Manager/pull/1194/commits/c4050dd1fc8d93e6f3f8068f3d7434dbadd1c045?

**Review by jom — APPROVED** [review]

**Comment by jom** [comment]
Snuck this into 1.29.0 :)

**Pull request merged**